sdk/fleet/Azure.ResourceManager.ContainerServiceFleet/src/autorest.md

```diff
@@ -77,4 +77,27 @@ acronym-mapping:
   URI: Uri
   Etag: ETag|etag
 
+operations-to-lro-api-version-override:
+  Fleets_CreateOrUpdate: "2016-03-30"
+  Fleets_Update: "2016-03-30"
+  Fleets_Delete: "2016-03-30"
+  FleetMembers_Create: "2016-03-30"
+  FleetMembers_Update: "2016-03-30"
+  FleetMembers_Delete: "2016-03-30"
+  UpdateRuns_CreateOrUpdate: "2016-03-30"
+  UpdateRuns_Delete: "2016-03-30"
+  FleetUpdateStrategies_CreateOrUpdate: "2016-03-30"
+  FleetUpdateStrategies_Delete: "2016-03-30"
+
+directive:
+- from: swagger-document
+  where: $.definitions.APIServerAccessProfile.properties.subnetId
```

Comment:
`subnetId` should always be a `arm-id`, what kind of other values does the service return?

Reply:
from our swagger manifest:
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/containerservice/resource-manager/Microsoft.ContainerService/fleet/preview/2023-08-15-preview/fleets.json#L1681C1-L1681C22

subnetid is a ResourceIdentifier type

currently, our Fleet code allows subnetid to be an empty string. but this will break ResourceIdentifier logic as it has a "nullorempty" check.
We have merged Fleet code to set subnetid to nil, which will fix this issue, but it has not rolled out yet.

this directive change should be temporary until the Fleet code has rolled out to all regions.

Reply:
Ok, then you can use `models-to-treat-empty-string-as-null` config to skip empty check, like this: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/autorest.md?plain=1#L348

Reply:
added this new logic